spec/controllers/teams_controller_spec.rb

```diff
@@ -233,78 +226,70 @@
     it 'succeeds when called as a team owner' do
       team = Team.create! valid_attributes
       team.owners << subject.current_user
-      team.members = team.members + team.owners
-      anotherUser = FactoryBot.create :user
+      another_user = FactoryBot.create :user
 
       expect {
-        post :assign_ownership, params: { id: team.id, team_member: anotherUser.id }
-      }.to change(TeamOwner, :count).by(1)
+        post :assign_ownership, params: { id: team.id, team_member: another_user.id }
+      }.to change(TeamUser, :count).by(1)
     end
 
 
     it 'fails when when not called as a team owner' do
       team = Team.create! valid_attributes
-      anotherUser = FactoryBot.create :user
+      another_user = FactoryBot.create :user
 
-      post :assign_ownership, params: { id: team.id, team_member: anotherUser.id }
+      post :assign_ownership, params: { id: team.id, team_member: another_user.id }
       expect(response).to be_forbidden
     end
 
     it 'should not add a team owner twice' do
```

Comment:
This test is somewhat senseless now, maybe it would be more useful to check if user is still owner after requesting `:assign_ownership` twice (to make sure there is no toggling happening)?